channels/OFF-CHAIN.md

```diff
@@ -291,23 +301,96 @@ different amounts, then that might communicate that it wants these instead.)
   blockchain to stop a cheater
 - `responder_pubkey` MUST be a valid ed25519 pubkey
 
+### `channel_reestabl`
```

Comment:
This causes anchor `#channel_reestabl` rather than `#channel_reestablish` in the outline.